client/src/components/glossary_components.tsx

```diff
@@ -11,27 +11,52 @@ import { is_a11y_mode } from "src/core/injected_build_constants";
 import { IconQuestion } from "src/icons/icons";
 import { glossary_href } from "src/link_utils";
 
-const GlossaryTooltipWrapper = ({ id, children, no_bottom_border }) =>
+interface GlossaryTooltipWrapperProps {
+  id: string;
+  children?: React.ReactNode;
+  no_bottom_border?: boolean;
+}
+
+interface GlossaryIconProps {
+  id: string;
+  alternate_text?: string;
+  arrow_selector?: any;
+  inner_selector?: any; // i don't know how to do the types for the selectors
+  icon_color?: string;
+  icon_alt_color?: string;
+}
+
+interface GlossaryItemProps {
+  id: string;
+  alternate_text: string;
+  item_class: string;
+}
+
+const GlossaryTooltipWrapper: React.FC<GlossaryTooltipWrapperProps> = ({
+  id,
+  children,
+  no_bottom_border,
+}) =>
   is_a11y_mode ? (
     <a
-      href={is_a11y_mode ? glossary_href(id) : null}
+      href={glossary_href(id)}
       title={is_a11y_mode ? trivial_text_maker("glossary_link_title") : null}
```

Comment:
Huh, strange. Given line 15, on lines 17 and 18 `is_a11y_mode` _must_ be true, don't think either of those ternaries were doing anything. Might as well drop the ternary on 18 (now 43) as well.